android/src/ti/barcode/BarcodeModule.java

```diff
@@ -219,6 +219,7 @@ private void populateYUVLuminanceFromRGB(int[] rgb, byte[] yuv420sp, int width,
 			}
 		}
 		hints.put(DecodeHintType.POSSIBLE_FORMATS, decodeFormats);
+		hints.put(DecodeHintType.ASSUME_GS1, null);
```

Comment:
@garymathews I guess, it will always show Symbology Identifier. What if developer do not want to use it? Can we expose it to developer as property?